### ISSUE TITLE
Telegram - Rewrite as a bot that posts the menu to a channel

### ISSUE THREAD

**Issue by BayMinimum** [BayMinimum/MealTweet#1]
BayMinimum/MealBotMax#1

@boghison Actual changes have to be made here, because of the strange anatomy..my bad;(

**Comment by cuzbog** [comment]
Is it ok if I rewrite the telegram bot to simply push the new menu out at a specified time daily? In that way, no changes have to be made here, I think (?)

**Comment by BayMinimum** [comment]
No, all the periodic stuffs happen here, because of free server quota. So, telegram_bot.js in this repo has to be edited. Currently, it sends private message to everyone. This part should be changed to push to the channel. (you may make use of a environment variable for specific information of channel and I'll add it)

**Comment by cuzbog** [comment]
Hmm... To be completely honest, the structure is a bit too convoluted for me to understand. However, I wrote an example of how it could be implemented in [boghison/MealBotMax](https://github.com/boghison/MealBotMax). You can check it out, if you want

**Comment by BayMinimum** [comment]
Ok, I've checked that. It seems your code cannot be applied directly to the current mechanics of the bot, but can you grant me to modify your code and use it if I attribute your code?

**Comment by cuzbog** [comment]
Yes, of course 💯
